### PR TITLE
Release notes

### PR DESCRIPTION
**Pull request by calebk1**
https://jira.corp.adobe.com/browse/MOB-16045

### CODE REVIEW

File: src/pages/documentation/release-notes/2021.md [hunk 1]
```diff
@@ -0,0 +1,759 @@
...
+* [flutter\_acpuserprofile](https://pub.dev/packages/flutter_acpuserprofile/versions/2.0.0)
+* [flutter\_acpanalytics](https://pub.dev/packages/flutter_acpanalytics/versions/2.0.0)
+* [flutter\_acpplaces](https://pub.dev/packages/flutter_acpplaces/versions/2.0.0)
+* [flutter\_acpplaces\_monitor](https://pub.dev/packages/flutter_acpplaces_monitor/versions/2.0.0)
+
+## June 8, 2021
+
+### iOS Campaign Standard 3.0.0
+
+* Initial release to support [Adobe Campaign Standard workflows](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/tree/master/using-mobile-extensions/adobe-campaign-standard) for Adobe Experience Platform Mobile SDKs for iOS in Swift. This extension library is [available as an open source project on Github](https://github.com/adobe/aepsdk-campaign-ios/).
```
Comment (emdobrin) [inline src/pages/documentation/release-notes/2021.md:437]: this link can be relative link, `../adobe-campaign-standard/index.md`

File: src/pages/documentation/release-notes/index.md [hunk 2]
```diff
@@ -0,0 +1,399 @@
+---
+description: Release notes and change logs for the Adobe Experience Platform Mobile SDKs.
+---
+
+# Release notes
+
```
Comment (emdobrin) [inline src/pages/documentation/release-notes/index.md:6]: Some content is missing here June 30 -> Aug 2

**Review by emdobrin — COMMENTED** [review]
Looks good, just few small comments

**Comment by emdobrin** [comment]
we also need to include these new files to `gatsby-config.js`

File: src/pages/documentation/release-notes/index.md (hunk 2 above)
Comment (calebk1) [reply src/pages/documentation/release-notes/index.md:6]: Yup! I'll re-add this once I do my last checkover time.

**Pull request merged**